worldedit-core/src/main/java/com/sk89q/worldedit/regions/Region.java

```diff
@@ -485,6 +477,23 @@ default IChunkSet processSet(IChunk chunk, IChunkGet get, IChunkSet set, boolean
         }
     }
 
+    private boolean isProcessExtra(IChunkSet set, boolean processExtra, int layer, DataArray arr) {
```

Comment:
shouldProcessExtra?

Reply:
Should probably actually call the method sometihng that implies it's doing something to the data at all